Source/SwiftLintFramework/Protocols/Rule.swift

```diff
@@ -23,6 +23,8 @@ extension Rule {
 
 public protocol OptInRule: Rule {}
 
+public protocol AutomaticTestableRule: Rule {}
```

Comment:
In an ideal world, this would be defined in the test target, but I thought that it'd be much more convenient to add the conformance when defining the rule than having to go to a `Rules+AutomaticTestable.swift` in the test target and add an extension there.